### PR TITLE
Update API Gateway tutorials

### PR DESCRIPTION
**Pull request by nataliasitko**
**Description**

Changes proposed in this pull request:

- Update API Gateway tutorials so they can be moved to Help Portal
- Add UI instructions for APIRule v2alpha1

**Pre-Merge Checklist**

- [ ] As a PR reviewer, verify code coverage and evaluate if it is acceptable.

**Related issues**

### CODE REVIEW

File: docs/user/tutorials/01-50-expose-and-secure-a-workload/01-51-get-jwt.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Get a JSON Web Token (JWT)
 
-This tutorial shows how to get a JSON Web Token (JWT), which can be used to access secured endpoints created in the [Expose and secure a workload with Istio](./01-53-expose-and-secure-workload-istio.md) and [Expose and secure a workload with JWT](./01-52-expose-and-secure-workload-jwt.md) tutorials.
+This tutorial shows how to get a JSON Web Token (JWT), which you can use to access secured endpoints created in the [Expose and secure a workload with Istio](./01-53-expose-and-secure-workload-istio.md) and [Expose and secure a workload with JWT](./01-52-expose-and-secure-workload-jwt.md) tutorials.
```
Comment (IwonaLanger) [inline docs/user/tutorials/01-50-expose-and-secure-a-workload/01-51-get-jwt.md:3]:
```suggestion
Learn how to get a JSON Web Token (JWT), which you can use to access secured endpoints created in the [Expose and secure a workload with Istio](./01-53-expose-and-secure-workload-istio.md) and [Expose and secure a workload with JWT](./01-52-expose-and-secure-workload-jwt.md) tutorials.
```

File: docs/user/tutorials/01-50-expose-and-secure-a-workload/01-51-get-jwt.md [hunk 2]
```diff
@@ -10,31 +10,24 @@ You use an OpenID Connect-compliant (OIDC-compliant) identity provider.
 
 1. Create an application in your OIDC-compliant identity provider. Save the client credentials: Client ID and Client Secret. 
 
-2. In the URL `https://{YOUR_IDENTITY_PROVIDER_INSTANCE}/.well-known/openid-configuration` replace `{YOUR_IDENTITY_PROVIDER_INSTANCE}` with the name of your OICD-compliant identity provider instance. Then, open the link in your browser. Save the values of the **token_endpoint**, **jwks_uri** and **issuer** parameters.
+2. In the URL `https://{YOUR_IDENTITY_PROVIDER_INSTANCE}/.well-known/openid-configuration` replace `{YOUR_IDENTITY_PROVIDER_INSTANCE}` with the name of your OICD-compliant identity provider instance. Then, open the link in your browser. Save the values of the **token_endpoint**, **jwks_uri**, and **issuer** parameters.
```
Comment (IwonaLanger) [inline docs/user/tutorials/01-50-expose-and-secure-a-workload/01-51-get-jwt.md:13]:
```suggestion
2. In the URL `https://{YOUR_IDENTITY_PROVIDER_INSTANCE}/.well-known/openid-configuration` replace `{YOUR_IDENTITY_PROVIDER_INSTANCE}` with the name of your OIDC-compliant identity provider instance. Then, open the link in your browser. Save the values of the **token_endpoint**, **jwks_uri**, and **issuer** parameters.
```

File: docs/user/tutorials/01-40-expose-workload/01-42-expose-workloads-multiple-namespaces.md [hunk 3]
```diff
@@ -3,15 +3,23 @@
...
+  > [!NOTE]
+  > Remember to enable automatic Istio sidecar proxy injection in each namespace. See [Enable Sidecar Injection for a Namespace](https://kyma-project.io/#/istio/user/tutorials/01-40-enable-sidecar-injection?id=enable-sidecar-injection-for-a-namespace).
+
+* [Set Up Your Custom Domain](../../01-10-setup-custom-domain-for-workload.md). Alternatively, you can use the default domain of your Kyma cluster and the default Gateway `kyma-system/kyma-gateway`.
+  
+  > [!NOTE]
+  > Because the default Kyma domain is a wildcard domain, which uses a simple TLS Gateway, it is recommended that you set up your custom domain for use in a production environment.
+
+  > [!TIP]
+  > To learn what the default domain of your Kyma cluster is, run `kubectl get gateway -n kyma-system kyma-gateway -o jsonpath='{.spec.servers[0].hosts}`.
```
Comment (IwonaLanger) [inline docs/user/tutorials/01-40-expose-workload/01-42-expose-workloads-multiple-namespaces.md:22]: Please indent lines 61-65, and change the point number to 4 in line 66. Sorry, I know that's not within the change you're making, but what's seen cannot be unseen ;) 
Screenshot 2024-12-31 at 13 20 57

File: docs/user/tutorials/01-50-expose-and-secure-a-workload/v2alpha1/01-53-expose-and-secure-workload-ext-auth.md [hunk 4]
```diff
@@ -5,85 +5,68 @@ Learn how to expose and secure services using APIGateway Controller and OAuth2.0
 
 ## Prerequisites
 
-* [Deploy a sample HTTPBin Service](../../01-00-create-workload.md).
-* [Set up your custom domain](../../01-10-setup-custom-domain-for-workload.md) or use a Kyma domain instead.
-* [Obtain a JSON Web Token (JWT)](../01-51-get-jwt.md).
+* You have a deployed workload.
+* To use CLI instructions, you must install [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) and [curl](https://curl.se/). Alternatively, you can use Kyma dashboard.
```
Comment (IwonaLanger) [inline docs/user/tutorials/01-50-expose-and-secure-a-workload/v2alpha1/01-53-expose-and-secure-workload-ext-auth.md:9]: There's no instruction for Kyma dashboard in the doc, so maybe it shouldn't be mentioned. Also, the heading in line 24 seems redundant.

File: docs/user/tutorials/01-30-set-up-mtls-gateway.md [hunk 5]
```diff
@@ -120,17 +119,16 @@ To expose a custom workload, create an APIRule. You can either use version `v1be
...
+3. Add the name `apirule-mtls`.
+4. Add a Gateway with the following values:
+   - **Namespace**: `default`
+   - **Gateway**: `kyma-mtls-gateway`
+5. Add the host `{SUBDOMAIN}.{DOMAIN_TO_EXPOSE_WORKLOADS}`.
+6. Add a rule with the following values:
+   - **Path**: `/.*`
+   - **Handler**: `no_auth`
+   - **Methods**: `GET`
+   - Select the name and port of your Service.
```
[the remark this answers is not in the document]
Comment (nataliasitko) [reply docs/user/tutorials/01-30-set-up-mtls-gateway.md:131]: Yes, it's part of the rule definition

File: docs/user/tutorials/01-40-expose-workload/01-42-expose-workloads-multiple-namespaces.md (hunk 3 above)
Comment (nataliasitko) [reply docs/user/tutorials/01-40-expose-workload/01-42-expose-workloads-multiple-namespaces.md:22]: sure, thank you

File: docs/user/tutorials/01-40-expose-workload/v2alpha1/01-41-expose-multiple-workloads.md [hunk 6]
```diff
@@ -1,150 +1,133 @@
...
+        port: {SECOND_SERVICE_PORT}
+EOF
+```
+<!-- tabs:end -->
 
 ## Access Your Workloads
-To access your HTTPBin Services, use [curl](https://curl.se).
 
-To call the endpoints, send `GET` requests to the HTTPBin Services:
+To call the endpoints, send `GET` requests to exposed the services:
```
[the remark this answers is not in the document]
Comment (nataliasitko) [reply docs/user/tutorials/01-40-expose-workload/v2alpha1/01-41-expose-multiple-workloads.md:126]: to the exposed services :)

**Review by IwonaLanger — APPROVED** [review]

**Pull request merged**